code/datums/emergency_calls/upp_commando.dm

```diff
@@ -51,3 +51,53 @@
 
 	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(to_chat), H, SPAN_BOLD("Objectives: [objectives]")), 1 SECONDS)
 
+/datum/emergency_call/upp_commando/lowthreat
```

Comment:
All of the typepaths that have "lowthreat" need to be snake_case'd so "low_threat"